src/components/oscal-utils/OSCALComponentResolver.js

```diff
@@ -1,5 +1,13 @@
 import OSCALResolveProfileOrCatalogUrlControls from "./OSCALProfileResolver";
 
+function processesToRun(componentDefinition) {
+  let count = 0;
+  componentDefinition.components.forEach((component) => {
+    count += component["control-implementations"]?.length ?? 0;
+  });
+  return count;
```

Comment:
```suggestion
  return componentDefinition.components.reduce((acc, component) => acc + component?.length ?? 0, 0);
```

Reply:
Now that it's so much simpler, we can just inline this rather than keeping it as a separate function call

Reply:
Or honestly even

```suggestion
  componentDefinition.components
  ?.map((component["control-implementations"]?.length ?? 0)
  ?.reduce((acc + len) => acc + len)
  || 0;
```

But that may be an extra iteration..